ScoutSuite/providers/azure/resources/virtualmachines/instances.py

```diff
@@ -92,8 +92,14 @@ async def _parse_instance(self, raw_instance):
             instance_dict['storage_profile']['OS Disk Size (GB)'] = raw_instance.storage_profile.os_disk.disk_size_gb
             instance_dict['storage_profile']['OS Disk Name'] = raw_instance.storage_profile.os_disk.name
             instance_dict['storage_profile']['OS Disk VHD'] = raw_instance.storage_profile.os_disk.vhd
-            instance_dict['storage_profile']['OS Managed Disk ID'] = raw_instance.storage_profile.os_disk.managed_disk.id.split('/')[-1]
-            instance_dict['storage_profile']['OS Managed Disk Storage Account Type'] = raw_instance.storage_profile.os_disk.managed_disk.storage_account_type
+            if raw_instance.storage_profile.os_disk.managed_disk is not None:
```

Comment:
This could also be evaluated as:

`if raw_instance.storage_profile.os_disk.managed_disk:`